test/component/Legend.spec.tsx

```diff
@@ -771,13 +790,13 @@ describe('<Legend />', () => {
                 animationDuration: 1500,
                 animationEasing: 'ease',
                 animationId: 0,
-                bottom: 5,
+                bottom: 65,
```

Comment:
unsure why all of these changed to be honest...

Reply:
Is it because the cleanup function from `mockHTMLElementProperty` is never called? Best to do that in `afterEach`. Check out the `restoreGetBoundingClientRect` and how it's used.

Reply:
fixed, oops. 

So in an `afterEach` doesn't work well unless we want to make the mock specific for offsetHeight and offsetWidth. I tried restoring the entire `HTMLElement.prototype` but that didn't seem to work. 

I'm sure there's a better way but not enough time to get there at least atm :P